lighthouse-core/report/html/templates.html

```diff
@@ -134,30 +132,15 @@
 <!-- Lighthouse score container -->
 <template id="tmpl-lh-scores-wrapper">
   <style>
-    .lh-scores-wrapper__background,
-    .lh-scores-wrapper__shadow {
-      position: absolute;
-      top: 0;
-      left: 0;
-      width: 100%;
-      height: 100%;
-      background: white;
-      border-radius: 8px;
-    }
-    .lh-scores-wrapper__shadow {
-      border-radius: 0;
-      box-shadow: rgba(0, 0, 0, 0.2) 0px 3px 0px -2px
-    }
     .lh-scores-container {
+      display: grid;
```

Comment:
😍

Reply:
do we actually use any grid properties though? what role does this play?

Reply:
It centered the thingy :P

Reply:
oh haha, there's probably a more obvious way to express centering :) 

we could do `display: flex` with `justify-content: center`?

Reply:
done